IDR/notebook/scripts/nbconverted/1.describe_screens.py

```diff
@@ -235,13 +328,50 @@ def describe_screen(screen_id):
 # In[10]:
 
 
+def collect_metadata(idr_name, values_list):
+    """Data collection and saving pipeline"""
+    session = requests.Session()
```

Comment:
This seems a bit strange to me defining inside the function. Maybe pass as an argument and instantiate outside?

Reply:
I did this because when using `Multiprocessing.Pool`, the `map` and `starmap` functions will only accept iterative arguments. I can add it to the list of tuples `[(idr_name, metadata_list), ..., (idr_name, metadata_list)]` that I pass into the function if that makes more sense.